controllers/ebulletin/index.js

```diff
@@ -17,30 +22,38 @@ const router = express.Router();
 function renderForm(req, res, data = null, errors = []) {
     res.render(path.resolve(__dirname, './views/ebulletin'), {
         title: req.i18n.__('toplevel.ebulletin.title'),
+        contactType: req.params.contactType,
         csrfToken: req.csrfToken(),
         validLocations: buildValidLocations(req.i18n),
+        validSectors: buildValidSectors(req.i18n),
         formValues: data,
         errors: errors,
     });
 }
 
 router
-    .route('/')
+    .route('/:contactType(stakeholder)?')
```

Comment:
Might suggest just splitting this into two separate route handlers, might be simpler compared to conditional params. But this works fine for now, so don't let this block you.